SU-HSDO/suhumsci
# feat (STN-472): Hero Height Refinement

# READY FOR REVIEW

## Summary
1. Reduce the height of hero elements in hero region and components (main content of page). The min-height of the image should match the height of the image in the carousel. Note that work for smaller screens for the carousel has not yet been completed. Therefore focus on large, desktop screen sizes.
2. Fixes issue where text overflows hero area.

Reference: [JIRA STN-472](https://sparkbox.atlassian.net/browse/STN-472)

Acceptance Criteria:
* Decrease the height of the hero by approximately 200px. Note that the carousel was recently reduced in height by this amount.
* Text area flexes with content from a minimum height and button styling
* Padding between elements should not change

## Need Review By (Date)
ASAP

## Urgency
high

## Steps to Test
1. In the CLI, run `npm run test` and confirm all our tests pass.
2. Go to http://sparkbox-sandbox.suhumsci.loc/qa/hero-height-refinements to review hero refinements. You will need to sync with the dev environment to grab the data to test this page. I did not add this page to staging because I was unsure of its status. The last we heard, staging was to be updated with data from prod after some testing with the smart date module. The hero is also found on the [home page](http://sparkbox-sandbox.suhumsci.loc/). 
Review the full-width hero at the top of the page AND the hero located within the main content of the page. Compare to examples on https://sparkbox-sandbox-dev.stanford.edu/hero-height-refinements to confirm the issue with text overflowing the hero area has been resolved.
- [x] Colorful theme hero on mobile
- [x] Colorful theme hero on desktop
- [x] Traditional theme hero on mobile
- [x] Traditional theme hero on desktop
3. Because the hero and carousel share some common markup, confirm the carousel in the colorful theme displays as expected. http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image Note: There is a card in the backlog to address traditional carousel refinements.
